styles/component/more-services/more-services.scss

```diff
@@ -0,0 +1,35 @@
+@import "@styles/baseConfig";
+
+// $navBarExpandAt: "screens.lg";
+
+@import "@styles/baseStyles";
+@import "@component/navbar";
+@import "@component/tooltip";
+@import "@component/footer";
+
+.more-services {}
```

Comment:
I created this class with page name and have given it to the page's body element to prevent
collisions of CSS classes in other pages. It'll scope the classes in this page. So please write the CSS that will only be used in this page inside this class